templates/rector.php.dist

```diff
@@ -12,4 +12,5 @@ __PATHS__
     // ->withPhpSets()
     ->withTypeCoverageLevel(0)
     ->withDeadCodeLevel(0)
-    ->withCodeQualityLevel(0);
+    ->withCodeQualityLevel(0)
+    ->withCodingStyleLevel(0);
```

Comment:
i think this is not needed for starter template for coding style

Reply:
Wasn't sure if it was needed or not, removed